src/components/member-page/member-data.js

```diff
@@ -33,12 +51,36 @@ const BottomDividerContainer = styled.div`
 const MemberData = ({
   role = { key: MEMBER_ROLE.explorer, name: '' },
   email = 'user@email.com',
-  joinDate = '2023/09/01',
+  joinDate = '2023/9/1',
   name = '',
+  articleReadCount = 0,
+  articleReadingTimeUnit = READING_TIME_UNIT.minute,
+  articleReadingTime = 0,
+  hideInfo = false,
 }) => {
   return (
     <MemberDataContainer>
       <H3 text={'個人資料'} />
+      {!hideInfo && (
+        <ReadingStatisticsContainer>
+          <ReadingStatistics>
+            <P1 text={'閱讀篇數'} />
+            <H1 text={articleReadCount.toLocaleString()} />
+            <P1 text={'篇'} />
+          </ReadingStatistics>
+          <ReadingStatistics>
+            <P1 text={'閱讀時間'} />
+            <H1
+              text={
+                articleReadingTime > 99999
+                  ? '99,999+'
+                  : articleReadingTime.toLocaleString()
```

Comment:
Could we specify the `locale` parameter for `toLocaleString` func ?

Reply:
fixed